### PR TITLE
[JENKINS-61302] - Remove the bogus "slave" reference in README

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Missed it during the previous scrubs. https://issues.jenkins-ci.org/browse/JENKINS-61302

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ Role Strategy plugin
 The Role Strategy plugin is meant to be used from [Jenkins](https://jenkins.io) to add a new role-based mechanism to manage users' permissions. 
 Supported features
 
-* Creating **global roles**, such as admin, job creator, anonymous, etc., allowing to set Overall, Slave, Job, Run, View and SCM permissions on a global basis.
+* Creating **global roles**, such as admin, job creator, anonymous, etc., allowing to set Overall, Agent, Job, Run, View and SCM permissions on a global basis.
 * Creating **project roles**, allowing to set only Job and Run permissions on a project basis.
```
Comment (nigelarmstrong-bc) [inline README.md]: This is called `project roles` in the documentation, but the UI calls it `Item roles`

File: README.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ Role Strategy plugin
...
 * Creating **agent roles**, allowing to set node-related permissions.
```
Comment (nigelarmstrong-bc) [inline README.md]: This is called `agent roles` in the documentation, but the UI calls it `Node roles`

File: README.md (hunk 1 above)
Comment (res0nance) [reply README.md:15]: https://github.com/jenkinsci/role-strategy-plugin/blob/623c95e6ba13cd9636fcad6e425cd41cb35f341d/src/main/resources/com/michelin/cio/hudson/plugins/rolestrategy/RoleStrategyConfig/manage-roles.jelly#L105-L110

I think you mean this, but this has no relation to the current PR

**Comment by oleg-nenashev** [comment]
I will merge it as is. I have not got to the further documentation cleanup, but I will make sure to report a follow-up issue for the terminology cleanup

**Pull request merged**